client/test_helpers.go

```diff
@@ -34,3 +38,24 @@ func (t TestAccountRetriever) GetAccountNumberSequence(_ NodeQuerier, addr sdk.A
 	}
 	return acc.Num, acc.Seq, nil
 }
+
+// CallCliCmd calls theCmd cobra command and returns the output in bytes.
+func CallCliCmd(clientCtx Context, theCmd func() *cobra.Command, extraArgs []string) ([]byte, error) {
```

Comment:
Provided that functions here are used only for testing purposes, I'd love to see this `test_helpers.go` files to live in their own packages somewhere, e.g. `testutil`